### ISSUE TITLE
pip install directory detected as DL3013

### ISSUE THREAD

**Issue by thetic** [hadolint/hadolint#389]
- [x] This is a bug report
- [ ] This is a feature request
- [x] I searched existing issues before opening this one

### Expected behavior
This shouldn't be DL3013 as it installs a setup.py in a directory
```
RUN pip install /app
```

### Actual behavior
```
/dev/stdin:1 DL3013 Pin versions in pip. Instead of `pip install <package>` use `pip install <package>==<version>`
```

### Steps to reproduce the behavior

```
echo "RUN pip install /app"  | hadolint -
```

**Output of `hadolint --version` or
  `docker run --rm hadolint/hadolint hadolint --version`:**

```bash
Haskell Dockerfile Linter v1.17.5-0-g443423c
```

**Comment by Ferroin** [comment]
The same issue exists when passing wheel files directly to `pip` for install.
